Registry creation error in fresh environment

<!--
Thank you for helping to improve KNDP!

Please be sure to search for open issues before raising a new one. We use issues
for bug reports and feature requests. 
-->

### What happened?
In fresh KNDP environment adding of registry return error:
```
panic: interface conversion: interface {} is nil, not []interface {}

goroutine 1 [running]:
github.com/kndpio/kndp/internal/registry.(*Registry).Create(0xc000a1db90?, {0x34f8258, 0xc0003ddd60}, 0xc000103860, 0x18c860?, 0xc000a22000?)
        /home/deep/www/kndp/kndp/internal/registry/registry.go:102 +0x386
github.com/kndpio/kndp/cmd/kndp/registry.(*createCmd).Run(0xc0006d8410, {0x34f8258, 0xc0003ddd60}, 0x2?, 0x2?, 0xc000bdbc88?)
        /home/deep/www/kndp/kndp/cmd/kndp/registry/create.go:54 +0x2aa
reflect.Value.call({0x2c4b660?, 0xc0006d8410?, 0x2fb1560?}, {0x2fb91cf, 0x4}, {0xc000a1e1e0, 0x4, 0x2f6f040?})
        /usr/local/go/src/reflect/value.go:596 +0xce7
reflect.Value.Call({0x2c4b660?, 0xc0006d8410?, 0x2cb80a0?}, {0xc000a1e1e0?, 0x2fb1560?, 0xc0006d25d0?})
        /usr/local/go/src/reflect/value.go:380 +0xb9
github.com/alecthomas/kong.callFunction({0x2c4b660?, 0xc0006d8410?, 0xc0009f5b90?}, 0x2fb7edb?)
        /home/deep/go/pkg/mod/github.com/alecthomas/kong@v0.8.1/callbacks.go:98 +0x40b
github.com/alecthomas/kong.(*Context).RunNode(0xc00021cc80, 0xc0009c6e10, {0x0, 0x0, 0x35155c8?})
        /home/deep/go/pkg/mod/github.com/alecthomas/kong@v0.8.1/context.go:765 +0x810
github.com/alecthomas/kong.(*Context).Run(0xc000a1e120?, {0x0?, 0xc0003ddd60?, 0x2acef80?})
        /home/deep/go/pkg/mod/github.com/alecthomas/kong@v0.8.1/context.go:790 +0x132
main.main()
        /home/deep/www/kndp/kndp/cmd/kndp/main.go:118 +0x54f
exit status 2

```
<!--
Please let us know what behaviour you expected and how KNDP diverged from
that behaviour.
-->


### How can we reproduce it?
1. Create new environment by `kndp env create local`
2. Create new registry by `kndp reg create`
<!--
Help us to reproduce your bug as succinctly and precisely as possible. Artifacts
such as example manifests or a script that triggers the issue are highly
appreciated!
-->

### What environment did it happen in?
KNDP version: 0.0.4-beta.4

<!--
Include at least the version or commit of KNDP you were running. Consider
also including your:

* Cloud provider or hardware configuration
* Kubernetes version (use `kubectl version`)
* Kubernetes distribution (e.g. Tectonic, GKE, OpenShift)
* OS (e.g. from /etc/os-release)
* Kernel (e.g. `uname -a`)
-->

--- a/internal/registry/registry.go
+++ b/internal/registry/registry.go
@@ -98,6 +98,12 @@ func (r *Registry) Create(ctx context.Context, client *kubernetes.Clientset, con
 
 	installer := configuration.GetManager(config, logger)
 	release, _ := installer.GetRelease()
+
+	if release.Config == nil {
+		release.Config = map[string]interface{}{
+			"imagePullSecrets": []interface{}{},
+		}
+	}
 	if release.Config["imagePullSecrets"] == nil {
 		release.Config["imagePullSecrets"] = []interface{}{}
 	}

